core/src/main/java/com/rim/IceField/Inventory.java

```diff
@@ -3,7 +3,9 @@
 import java.util.ArrayList;
 
 public class Inventory {
-    protected ArrayList<ItemBase> items; //items in the inventory of each player
+
+    //had to do it static because belongs to the class
+    protected static ArrayList<ItemBase> items; //items in the inventory of each player
```

Comment:
this attribute is not static, each player has their own inventory where they collect the items, don't they?